src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -378,6 +378,7 @@ static PrepareMethod valueOfString(String value) throws SQLServerException {
     SELECT_METHOD("selectMethod", "direct"),
     DOMAIN("domain", ""),
     SERVER_NAME("serverName", ""),
+    IPADDRESSPREFERENCE("IPAddressPreference", "IPv4First"),
```

Comment:
[nitpick] IPADDRESS_PREFERENCE seems a bit easier to read?

Reply:
Agreed, changes made.